leaflet-geoman.d.ts

```diff
@@ -1,340 +1,811 @@
+
+
+        /**
+         * PM map interface.
+         */
+        interface PMMap extends PMDrawMap, PMEditMap, PMDragMap, PMRemoveMap, PMCutMap {
+
+            Toolbar: PMMapToolbar;
+
+            /** Adds the Toolbar to the map. */
+            addControls(options?: ToolbarOptions): void;
+
+            /** Toggle the visiblity of the Toolbar. */
+            removeControls(): void;
+
+            /** Returns true if the Toolbar is visible on the map. */
+            controlsVisible(): boolean;
+
+            /** Toggle the visiblity of the Toolbar. */
+            toggleControls(): void;
+
+            setLang(
+                lang:
+                    | 'da'
+                    | 'de'
+                    | 'el'
+                    | 'en'
+                    | 'es'
+                    | 'fa'
+                    | 'fr'
+                    | 'hu'
+                    | 'id'
+                    | 'it'
+                    | 'nl'
+                    | 'no'
+                    | 'pl'
+                    | 'pt_br'
+                    | 'ro'
+                    | 'ru'
+                    | 'sv'
+                    | 'tr'
+                    | 'ua'
+                    | 'zh'
+                    | 'zh_tw',
+                customTranslations?: Translations,
+                fallbackLanguage?: string
+            ): void;
+
+            /** Set globalOptions and apply them. */
+            setGlobalOptions(options: GlobalOptions): void;
+
+            /** Apply the current globalOptions to all existing layers. */
+            applyGlobalOptions(): void;
+
+            /** Returns the globalOptions. */
+            getGlobalOptions(): GlobalOptions;
+        }
+
+        class Translations {
+            tooltips?: {
+                placeMarker?: string;
+                firstVertex?: string;
+                continueLine?: string;
+                finishLine?: string;
+                finishPoly?: string;
+                finishRect?: string;
+                startCircle?: string;
+                finishCircle?: string;
+                placeCircleMarker?: string;
+            };
+
+            actions?: {
+                finish?: string;
+                cancel?: string;
+                removeLastVertex?: string;
+            };
+
+            buttonTitles?: {
+                drawMarkerButton?: string;
+                drawPolyButton?: string;
+                drawLineButton?: string;
+                drawCircleButton?: string;
+                drawRectButton?: string;
+                editButton?: string;
+                dragButton?: string;
+                cutButton?: string;
+                deleteButton?: string;
+                drawCircleMarkerButton?: string;
+            };
+        }
+
+        type ACTION_NAMES = 'cancel' | 'removeLastVertex' | 'finish' | 'finishMode';
+
+        class Action {
+            text: string;
+            onClick?: (e: any) => void;
+        }
+
+        type TOOLBAR_CONTROL_ORDER =
```

Comment:
"I'm not sure if this line is correct: `changeControlOrder(order: 'drawCircle' | 'drawRectangle' | 'removalMode' | 'editMode'[]): void;` It can be each one of the "modes" :` drawMarker, drawCircleMarker, drawPolyline, drawRectangle, drawPolygon, drawCircle, editMode, dragMode, cutPolygon, removalMode` and every mode that is created with `createCustomControl` or `copyDrawControl` -> `CustomControlOptions.name`"

Like this?